std/experimental/ndslice/slice.d

```diff
@@ -46,7 +46,7 @@ Returns:
 auto sliced(
     Flag!"replaceArrayWithPointer" replaceArrayWithPointer = Yes.replaceArrayWithPointer,
     Flag!"allowDownsize" allowDownsize = No.allowDownsize,
-    Range, Lengths...)(Range range, Lengths lengths)
+    Range, Lengths...)(Range range, Lengths lengths)  @trusted
```

Comment:
Why does this function need to be `@trusted`? We should be making the function that it calls `@safe` instead.